client/src/utils/stringCorrections.ts

```diff
@@ -1,11 +1,21 @@
-export const isInputValid = event => {
+export const isInputValid = (event: string): boolean => {
   return /(^[A-Za-z]\s{1}}[A-Za-z])*$/.test(event);
 };
 
-export const formatNameForUrl = name => {
+export const formatNameForUrl = (name: string): string => {
   return name.replace(/\s+/g, '');
 };
 
-export const resetLocationName = name => {
+export const separateCapitalizeName = (name: string): string => {
   return name.replace(/([a-z])([A-Z])/g, '$1 $2');
 };
+export const capitalizeFirstLetter = (name: string): string => {
+  return name.substring(0, 1).toUpperCase() + name.substring(1).replace(/([a-z])([A-Z])/g, '$1 $2');
```

Comment:
`.replace(/([a-z])([A-Z])/g, '$1 $2')` was used in `separateCapitalizeName` function, so please reuse it here